WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -839,6 +839,12 @@ import WordPressFlux
                                         })
     }
 
+    fileprivate func toggleSavedForLater(for post: ReaderPost) {
+        SaveForLaterAction().execute(with: post, context: managedObjectContext(), completion: { [weak self] in
+            self?.syncHelper.syncContent()
```

Comment:
Do we need to sync content currently if save for later is local only?

Reply:
I have no idea, to be honest, but I'll remove it for now. What does `syncContent` do exactly?